cmd/cluster-agent/custommetrics/server.go

```diff
@@ -128,6 +128,10 @@ func (a *DatadogMetricsAdapter) Config() (*apiserver.Config, error) {
 		// Ensure backward compatibility. 443 by default, but will error out if incorrectly set.
 		// refer to apiserver code in k8s.io/apiserver/pkg/server/option/serving.go
 		a.SecureServing.BindPort = config.Datadog.GetInt("external_metrics_provider.port")
+		// Default in External Metrics is TLS 1.2
+		if !config.Datadog.GetBool("cluster_agent.allow_legacy_tls") {
+			a.SecureServing.MinTLSVersion = "VersionTLS13"
```

Comment:
can we use a const string here for the value? 
is it the same value than `tls.VersionTLS13`

Reply:
It's the same value but for some reason they decided that it should be inputted as a string and then remap to `tls.VersionTLS13` 🤷‍♂️